content/plugins/fabric.md

```diff
@@ -238,3 +237,47 @@ Using a tasks file instead of a list of commands will allow you to use python co
 Using a list of commands might be a good solution for very simple cases in which you wouldn't want to maintain a tasks file.
 
 {{% /gsTip %}}
+
+
+# ctx for fabric-plugin
+
+Starting with Cloudify 3.4 and fabric-plugin 1.4, we now support using ctx in Python scripts executed by the fabric plugin on remote machines. Most of the functionality is similiar to how the script plugin exposes the ctx object.
+
+## Executing ctx commands
+
+Previously, to use the fabric plugin to execute Python scripts you had to use ctx commands like so:
+
+{{< gsHighlight  python  >}}
+os.system('ctx logger info Hello!')
+{{< /gsHighlight >}}
+
+Instead, you can now do one of two things to achieve the same result:
+
+{{< gsHighlight  python  >}}
+from cloudify import ctx
+
+ctx.logger.info("Hello!")
+{{< /gsHighlight >}}
+
+or 
+
+{{< gsHighlight  python  >}}
+from cloudify import ctx
+
+ctx('logger info Hello!')
+{{< /gsHighlight >}}
+
+The first example shows native ctx usage which can be used to perform most of the trivial actions you can perform using the script plugin like using the logger; retrieving runtime properties and setting them for both node instances and relationship instances; retrieving node properties; downloading blueprint resources; aborting operations, and more. 
```

Comment:
you don't set runtime properties of relationship instances but rather you set the source/target node instances runtime properties in relationship operations